<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPort_testClass</class>
 <widget class="QMainWindow" name="SerialPort_testClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calibrator App</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/icon.png</normaloff>Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Offline</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ReceiveDataLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive data:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BreakButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Break</string>
    </property>
   </widget>
   <widget class="QLabel" name="ControlProcesslabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Control process:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="DriversWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="Driver1">
     <attribute name="title">
      <string>Driver 1</string>
     </attribute>
     <widget class="QSpinBox" name="TargetPositionSpinBox1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
     <widget class="QLabel" name="TargetPositionLabel_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Target position:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Driver2">
     <attribute name="title">
      <string>Driver 2</string>
     </attribute>
     <widget class="QLabel" name="TargetPositionLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Target position:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="TargetPositionSpinBox2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Driver3">
     <attribute name="title">
      <string>Driver 3</string>
     </attribute>
     <widget class="QLabel" name="TargetPositionLabel_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Target position:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="TargetPositionSpinBox3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="ControlPositionFromPCcheckBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set position from PC</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="RunVectorButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Vector </string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextPositionButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadFileButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplySetting1">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply settings</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxStepSize1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>221</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="StepSizelabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Step Size:</string>
    </property>
   </widget>
   <zorder>StartButton</zorder>
   <zorder>label1</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>ReceiveDataLabel</zorder>
   <zorder>BreakButton</zorder>
   <zorder>ControlProcesslabel</zorder>
   <zorder>DriversWidget</zorder>
   <zorder>ControlPositionFromPCcheckBox</zorder>
   <zorder>statusLabel</zorder>
   <zorder>RunVectorButton</zorder>
   <zorder>NextPositionButton</zorder>
   <zorder>LoadFileButton</zorder>
   <zorder>ApplySetting1</zorder>
   <zorder>spinBoxStepSize1</zorder>
   <zorder>StepSizelabel</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SerialPort_test.qrc"/>
 </resources>
 <connections/>
</ui>
